spec/integ/admin-rooms.spec.js

```diff
@@ -14,7 +14,7 @@ describe("Creating admin rooms", function() {
     var botUserId = "@" + appConfig.botLocalpart + ":" + appConfig.homeServerDomain;
 
     beforeEach(function(done) {
-        test.beforeEach(this, env);
+        test.beforeEach(this, env); // eslint-disable-line no-invalid-this
```

Comment:
Is there a way to just disable these checks for the whole spec directory? Maybe a separate config file?